cmake/onnxruntime.cmake

```diff
@@ -72,6 +72,10 @@ target_link_libraries(onnxruntime PRIVATE
     onnxruntime_mlas
     ${onnxruntime_EXTERNAL_LIBRARIES})
 
+if (onnxruntime_ENABLE_LANGUAGE_INTEROP_OPS)
+  target_link_libraries(onnxruntime PRIVATE onnxruntime_language_interop onnxruntime_pyop)
```

Comment:
It's not correct. In this way, you lose the control of the ordering the libraries.  onnxruntime_language_interop  shouldn't be after ${onnxruntime_EXTERNAL_LIBRARIES}. Your build will fail because of this.

Reply:
Build not failing but fails without this.